app/assets/stylesheets/admin/batches.css

```diff
@@ -103,7 +104,7 @@
   margin-left: 95%;
   border-bottom: 5px solid;
   border-right: 5px solid;
-  transform: rotate(-45deg);
+  transform: rotate(-135deg);
```

Comment:
Would this work better (so that the arrow appears down when not expanded, and up when it's expanded)? 

`transform: rotate(45deg);`

And on line 112:   `transform: rotate(-135deg);`

Reply:
```suggestion
transform: rotate(45deg);
```